aws/kms.go

```diff
@@ -0,0 +1,51 @@
+package aws
+
+import (
+	"encoding/base64"
```

Comment:
Can you use `github.com/hairyhenderson/gomplate/base64` instead? Keeps things consistent (especially with the `Decode` case)

Reply:
Yup, I was actually debating if I should use your implementation instead. Makes sense! 😄